README.md

```diff
@@ -52,10 +52,13 @@ Uptime Kuma is now running on http://localhost:3001
 
 Requirements:
 
-- Platform
-  - ✅ Major Linux distros such as Debian, Ubuntu, CentOS, Fedora and ArchLinux etc.
-  - ✅ Windows 10 (x64), Windows Server 2012 R2 (x64) or higher
-  - ❌ Replit / Heroku
+| Platform | Supported |
+|-|-|
+| Major Linux distros such as Debian, Ubuntu, CentOS, Fedora and ArchLinux etc. | :white_check_mark: |
+| Windows 10 (x64), Windows Server 2012 R2 (x64) or higher | :white_check_mark: |
+| Replit / Heroku | ⚠️ Deprecated |
```

Comment:
first of all, I would prefer that we don't change the wording of ❌ -> ⚠️, as this would imply that this was ever supported imo.

idk if we should link the issues

```suggestion
| Heroku | ❌ Not supported, see https://github.com/louislam/uptime-kuma/issues/272#issuecomment-907087708 |
| Replit | ❌ Not supported, see https://github.com/louislam/uptime-kuma/issues/3129#issuecomment-1574770233 |
```

Reply:
Yes, it should be ❌. 

I think the link is not necessary and Heroku / Replit can be in the same line. Technically there are not OS, but I just don't want people to keep opening a new issue of those platforms.

Reply:
> Yes, it should be ❌.
> 
> I think the link is not necessary and Heroku / Replit can be in the same line. Technically there are not OS, but I just don't want people to keep opening a new issue of those platforms.

I don't think it should be ❌ since i run uptime kuma on Replit and it works just fine!

Reply:
Please search old issues, there are a lot of strange reports. I don't want to deal with them.

Reply:
I understand you don't want to deal with them! I will update it!